### PR TITLE
zh-translation: /docs/ops/deployment/performance-and-scalability/index.md

### PR DESCRIPTION
**Pull request by ilylia**
ref: https://github.com/servicemesher/istio-official-translation/issues/1658

### CODE REVIEW

File: content/zh/docs/ops/deployment/performance-and-scalability/index.md [hunk 1]
```diff
@@ -73,15 +73,15 @@ Pilot 根据用户编写的配置文件和系统当前状态来配置 sidecar 
 
 响应被返回给客户端后，Envoy 代理将收集原始的遥测数据。为请求收集原始遥测数据所花费的时间并没有统计在完成该请求所需的总时间里。但是，worker 在忙着处理请求时是不会立刻开始处理下一个请求的。此过程会增加下一个请求的队列等待时间并影响平均延迟和尾部延迟。实际的尾部延迟取决于流量模式。
 
-在网格内部，一个请求会先遍历客户端代理，然后遍历服务端代理。在90%的情况下，数据路径上的这两个代理每 1000 QPS 会产生 7ms 的延迟。
+在网格内部，一个请求会先遍历客户端代理，然后遍历服务端代理。在90%的情况下，数据路径上的这两个代理每 1000 QPS 会产生 6.3 ms 的延迟。
```
Comment (yuxiaobo96) [inline content/zh/docs/ops/deployment/performance-and-scalability/index.md]:
```suggestion
在网格内部，一个请求会先遍历客户端代理，然后遍历服务端代理。在 90% 的情况下，数据路径上的这两个代理每 1000 QPS 会产生 6.3 ms 的延迟。
```

Comment (ilylia) [reply content/zh/docs/ops/deployment/performance-and-scalability/index.md]: fixed

**Comment by yuxiaobo96** [comment]
@rootsongjc 
已修改，可以合并了

**Pull request merged**